xmtp/Cargo.toml

```diff
@@ -36,8 +38,12 @@ ethers-core = "2.0.4"
 prost = { version = "0.11", features = ["prost-derive"] }
 futures = "0.3.28"
 base64 = "0.21.1"
-tokio = "1.28.1"
+tokio = { version = "1.32.0", features = ["full"] }
 anyhow = "1.0.71"
+x25519-dalek = "2.0.0"
```

Comment:
Added a bunch of the underlying cryptography crates that we are already using in Vmac.